recipe/meta.yaml

```diff
@@ -30,20 +30,20 @@ outputs:
         - setuptools
         - wheel
         - packaging
-        - param 1.9.0
+        - param >=1.9.0,<3.0
```

Comment:
```suggestion
        - param =1.9
```

Reply:
`param >=1.9.0` is enough

Reply:
Is `param =1.9` equivalent to `param >=1.9.0`?

Reply:
param =1.9 is the same as param 1.9
param >=1.9.0 will resolve to the latest param available.

What the linter tries to highlight is that when linking to C libraries, we want to link to the lowest compatible version, to ensure maximum range compatibility. For other type of libraries, it doesn't matter as much.